server/modules/user/service/userService.js

```diff
@@ -51,7 +53,7 @@ const inviteUser = async (user, surveyId, email, groupUuid) => {
   }
 }
 
-const updateUser = async (user, surveyId, userUuid, name, email, groupUuid) => {
+const updateUser = async (user, surveyId, userUuid, name, email, groupUuid, fileReq) => {
```

Comment:
why Req suffix to fileReq param? Service might not necessarily know that the file passed come from the request

Reply:
Renamed to "file"